demo/shared/modules/react-page-transition/PageTransition.module.css

```
@@ -0,0 +1,22 @@
+.fade {
+    &.enter {
+        opacity: 0;
+    }
+
+    &.enterActive,
+    &.enterDone {
+        opacity: 1;
+        transition: opacity 0.6s;
+        transition-delay: 0.3s;
```

nitpick: can be done in a single line 
```suggestion
        transition: opacity 0.6s 0.3s;
```

---

✅